docs/RC/GetInteriorVehicleData/index.md

```diff
@@ -39,6 +41,13 @@ If the parameter `subscribe` is set to true, the mobile application has requeste
 HMI must provide optional `moduleId` param  in GetInteriorVehicleData_response.
 HMI must return in GetInteriorVehicleData_response the current value of the display mode used in HMI if `moduleType = HMI_SETTINGS` .
 
+!!! NOTE
+
+If during resumption HMI responds with error to GetInteriorVehicleData request SDL reverts already subscribed data and fails resumption, removing information about this subscription.  
```

Comment:
If during resumption HMI responds with error to GetInteriorVehicleData request or responds with SUCCESS to GetInteriorVehicleData  but with parameter isSubscribed=false, SDL reverts already subscribed data and fails resumption for related application(s), removing information about this subscription.

Reply:
thank you, done in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/225/commits/9074c7d61dea3f16ee213393369743b367f7b50d